chromium_src/chrome/elevation_service/elevator.cc

```diff
@@ -28,6 +28,34 @@
 namespace elevation_service {
 
 HRESULT Elevator::InstallVPNServices() {
+  HRESULT hr = ::CoImpersonateClient();
```

Comment:
please add a comment explaining the purpose of this (verifying that it can only be called from an exe in C:\Program Files)

Reply:
Updated!